.env.sample

```diff
@@ -49,6 +49,12 @@ SMTP_USERNAME=
 SMTP_PORT=
 SMTP_SSL_TLS=
 
+# Enable or disable the storage of logs
+LOG=false
+
+
+# Path of file that will store logs
+LOG_PATH=./logs/transaction.log
```

Comment:
Update the `.gitignore` file. Add:

1. The `logs/` directory and all files in it. There are examples for the `images/` directory
2. All files ending in `.log` 

We don't wan these being added to the repo

Reply:
I added a `logs/README.md` file as suggested by @EshaanAgg. I've changed the `.gitignore` file to ignore all files inside the `logs` dir except README.md.
And, ignoring all the `.log` files.